<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainToDoPanel</class>
 <widget class="QMainWindow" name="MainToDoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="sideFixedMenu" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>278</width>
      <height>659</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(33, 33, 33);
}
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="userAccountIcon">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;
background: transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Image/Icons/user_account.png</normaloff>
               <disabledoff>:/Image/Icons/user_account.png</disabledoff>:/Image/Icons/user_account.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="userInfoLB">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	font: 10pt &quot;Playwrite DE Grund&quot;;
}</string>
             </property>
             <property name="text">
              <string>amirchami1383@gmail.com</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	text-align : left;
	background : transparent; 
	border-radius : 5 px;
	font: 9pt &quot;Playwrite DE Grund&quot;;
	color: rgb(255, 255, 255);
	height : 40 px;
	padding-left:5px;
	border-radius:5px;
}
QPushButton:hover {
	background-color: rgb(51, 51, 51);
	font: 700 9pt &quot;Playwrite DE Grund&quot;;
}
QPushButton:checked {
	background-color: rgb(51, 51, 51);
	font: 700 9pt &quot;Playwrite DE Grund&quot;;
}
QScrollArea {
	border : 1px solid rgb(51, 51, 51);
	border-radius : 5 px;
}</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>520</height>
             </rect>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>7</x>
               <y>10</y>
               <width>240</width>
               <height>190</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="myDayPB">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string> My Day</string>
                </property>
                <property name="icon">
                 <iconset resource="Resources.qrc">
                  <normaloff>:/Image/Icons/out_line_sun.png</normaloff>
                  <normalon>:/Image/Icons/fill_sun.png</normalon>
                  <disabledon>:/Image/Icons/fill_sun.png</disabledon>:/Image/Icons/out_line_sun.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="importantPB">
                <property name="text">
                 <string> Important</string>
                </property>
                <property name="icon">
                 <iconset resource="Resources.qrc">
                  <normaloff>:/Image/Icons/out_line_star.png</normaloff>
                  <normalon>:/Image/Icons/fill_star_color.png</normalon>:/Image/Icons/out_line_star.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="assignedToMePB">
                <property name="text">
                 <string> Assigned to me</string>
                </property>
                <property name="icon">
                 <iconset resource="Resources.qrc">
                  <normaloff>:/Image/Icons/out_line_user.png</normaloff>
                  <normalon>:/Image/Icons/fill_user_color.png</normalon>:/Image/Icons/out_line_user.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="tasksPB">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string> Tasks</string>
                </property>
                <property name="icon">
                 <iconset resource="Resources.qrc">
                  <normaloff>:/Image/Icons/home_2.png</normaloff>
                  <normalon>:/Image/Icons/fill_home.png</normalon>:/Image/Icons/home_2.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="newListPB">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	text-align : left;
	background : transparent; 
	border-radius : 5 px;
	font: 9pt &quot;Playwrite DE Grund&quot;;
	color: rgb(255, 255, 255);
	height : 40 px;
	padding-left:5px;
	border-radius:5px;
}
QPushButton:hover {
	background-color: rgb(51, 51, 51);
	font: 700 9pt &quot;Playwrite DE Grund&quot;;
}</string>
         </property>
         <property name="text">
          <string> New list</string>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/Image/Icons/icons8-plus-96.png</normaloff>:/Image/Icons/icons8-plus-96.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="mainWindow" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>651</width>
      <height>659</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(33, 33, 33);
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="titleWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="titleLB">
              <property name="styleSheet">
               <string notr="true">QLabel {
	text-align : left;
	background : transparent; 
	border-radius : 5 px;
	font: 12pt &quot;Playwrite DE Grund&quot;;
	color: rgb(255, 255, 255);
	height : 40 px;
	padding-left:5px;;
}
</string>
              </property>
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>308</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="mainStack">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(28, 28, 28);</string>
         </property>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
    <zorder>titleWidget</zorder>
    <zorder>mainStack</zorder>
   </widget>
   <widget class="QWidget" name="sideTasksMenu" native="true">
    <property name="geometry">
     <rect>
      <x>985</x>
      <y>1</y>
      <width>105</width>
      <height>87</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(33, 33, 33);
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>mainWindow</zorder>
   <zorder>sideFixedMenu</zorder>
   <zorder>sideTasksMenu</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
